
# HG changeset patch
# User Manu Abraham <abraham.manu@gmail.com>
# Date 1265828497 -14400
# Node ID 943c1b03aea35d63e3f21e6b3010caf308c540d9
# Parent  e994c0877f9b115fcf44280d8ac9c157ee99f329
czap: use %m modifier in sscanf instead of %a

%a is a old glibc extension and conflicts with
the floating point modifier %a in C99

From: Janne Grunau <j@jannau.net>

--- a/util/szap/czap.c	Wed Feb 10 22:58:12 2010 +0400
+++ b/util/szap/czap.c	Wed Feb 10 23:01:37 2010 +0400
@@ -143,7 +143,7 @@
 	}
 	printf("%3d %s", chan_no, chan);
 
-	if ((sscanf(chan, "%a[^:]:%d:%a[^:]:%d:%a[^:]:%a[^:]:%d:%d\n",
+	if ((sscanf(chan, "%m[^:]:%d:%m[^:]:%d:%m[^:]:%m[^:]:%d:%d\n",
 				&name, &frontend->frequency,
 				&inv, &frontend->u.qam.symbol_rate,
 				&fec, &mod, vpid, apid) != 8)

